tests/utest

```diff
@@ -428,16 +428,13 @@ fi
 # utility functions in rt_utils need to be able to see variables in utest
 source rt_utils.sh
 
+source $PATHTR/NEMS/src/conf/module-setup.sh.inc
 # Machine-dependent libraries, modules, variables, etc.
 if [[ $MACHINE_ID = hera.* ]]; then
 
-  source $PATHTR/NEMS/src/conf/module-setup.sh.inc
-  module use $PATHTR/modulefiles/${MACHINE_ID}
-  module load fv3
```

Comment:
I am curious, do we not need to load the module file when running utest on hera?

Reply:
Similar to rt.sh, modulefiles are loaded by compile.sh and run_test.sh.